lib/main.js

```diff
@@ -259,7 +259,9 @@ var loadConfig_p = qutil.serialized(function() {
     if (prots.length === 0){
       throw new Error("All protocols were disabled. You must leave at least one protocol enabled.");
     }
-    logger.info("Disabled protocols: " + configRouter.disabledProtocols.join(","));
+    if (configRouter.disabledProtocols && configRouter.disabledProtocols.length > 0){
+      logger.info("Disabled protocols: " + configRouter.disabledProtocols.join(","));
```

Comment:
I'm OK with this PR, but maybe it's more interesting to log the enabled protocols?